pom.xml

```diff
@@ -62,11 +63,13 @@
         <module>rubix-client</module>
         <module>rubix-common</module>
         <module>rubix-presto</module>
+        <module>rubix-prestosql</module>
         <module>rubix-hadoop2</module>
         <module>rubix-bookkeeper</module>
         <module>rubix-tests</module>
         <module>assembly</module>
         <module>rubix-rpm</module>
+        <module>rubix-shaded</module>
```

Comment:
module name could be more specific i.e., **rubix-prestosql-shaded**

Reply:
I avoided that because prestodb version will be deprecated slowly. But prestosql here would avoid confusion about the contents, will refactor.

Reply:
I thought I had changed it, my comments was thinking it is named rubix-presto-shaded. Will rename to that.